docs-chef-io/content/server/v14/config_rb_server_optional_settings.md

```diff
@@ -972,6 +972,11 @@ This configuration file has the following settings for `oc_chef_authz`:
 :   The amount of time (in milliseconds) to wait for a connection to be
     established. Default value: `'[{connect_timeout, 5000}]'`.
 
+`oc_chef_authz['max_connection_request_limit']`
+
+:   The max number of requests allowed per connections. 
+    Default value: `100`.
```

Comment:
I have been unable to find any info on this ibrowse setting, or what it means.  Do you have a link to any docs that explain the setting?  I can't even find the setting in the repository:
```
$ git clone git@github.com:chef/ibrowse
Cloning into 'ibrowse'...
remote: Enumerating objects: 1535, done.
remote: Counting objects: 100% (13/13), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 1535 (delta 6), reused 9 (delta 3), pack-reused 1522
Receiving objects: 100% (1535/1535), 968.93 KiB | 4.61 MiB/s, done.
Resolving deltas: 100% (990/990), done.
$ cd ibrowse/
$ git grep max_connection_request_limit
$
```

Reply:
This is in git@github.com:chef/opscode_httpc repo, Could you please verify it there